packages/neuron-wallet/src/utils/scriptAndAddress.ts

```diff
@@ -1,6 +1,27 @@
-import { Script } from '@ckb-lumos/base'
+import { key } from '@ckb-lumos/hd'
+import { type Script } from '@ckb-lumos/base'
 import { predefined } from '@ckb-lumos/config-manager'
 import { encodeToAddress, parseAddress } from '@ckb-lumos/helpers'
+import { systemScripts } from './systemScripts'
+
+export enum DefaultAddressNumber {
+  Change = 10,
+  Receiving = 20,
+}
+
+export const Ox = (hex: string) => (hex.startsWith('0x') ? hex : `0x${hex}`)
```

Comment:
Do not name a variable starting with `digit`, should be started with a letter, `_` or `$`

Ref: https://javascript.info/variables#variable-naming

Reply:
The first letter of `Ox` is the uppercase letter `'O'`, because variable names can't start with digits.

This naming is for easy writing and understanding.
```ts
Ox(accountKeychain.chainCode.toString('hex'))
```

Reply:
- This is quite misleading...

Reply:
How about naming it `prefixWith0x` or `add0xPrefix`

Reply:
Renamd `Ox()` to `prefixWith0x()`

Reply:
I actually think `Ox` is not bad because it's short and easy to understand. Just like the `t` method.
```ts
import { t } from 'i18next'
```

Reply:
> I actually think `Ox` is not bad because it's short and easy to understand. Just like the `t` method.
> 
> ```ts
> import { t } from 'i18next'
> ```

It relies on `font` and `Editor/Viewer` to differentiate `0` and `O` and increase the proneness to typo. `t` for `translation` doesn't have this case.